app/components/view-section-table-row/macro.njk

```diff
@@ -2,6 +2,11 @@
 
   {% if params.layout %}
     {% set layout = params.layout | lower %}
+    {% if params.innerComponentClass %}
```

Comment:
I think you will have to set this in the horizontal too as otherwise if params.innerComponentClass is passed in but params.layout is not then `innerComponentClass` will not get set

otherwise call it something specific to the vertical layout like `verticalInnerComponentClass`